packages/EasyDecision/src/Interfaces/DecisionContextInterface.php

```diff
@@ -4,7 +4,7 @@
 
 namespace EonX\EasyDecision\Interfaces;
 
-interface ContextInterface
+interface DecisionContextInterface
```

Comment:
The objective of this PR is to add Symfony Web Profiler support for this package, that shouldn't require any naming changes to the existing classes AND changing the name of an interface like that is a BC break so it should be done either safely by supporting both name until next major version or simply wait for the next major version.

What happen if a project is using this interface as a parameter type hint because they're using the decision context in their code? It will break. Let's revert that and if we really want to change it then it should be its own PR.